### DISCUSSION TITLE
Is there a plan to support Android 14?🥺

### DISCUSSION THREAD

**Discussion by Furret162** [RocketChat/Rocket.Chat.ReactNative#5806] [Q&A] [answered]
Hello. I am using RocketChat on Android 14. Is there a plan for RocketChat to support Android 14?
Google has stated that Android 14 support should be in place by August 31. When will RocketChat support Android 14?
https://developer.android.com/google/play/requirements/target-sdk

**Comment by diegolmello** [comment, marked as answer]
It's already supported https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/0d1a911a7c51c77302e8c73361a23071adad2311/android/build.gradle#L9

**Reply by Furret162** [reply]
sorry, I overlooked it. Thank you😊
